pkg/service/src/resolvers/query/displayValues/data.json

```diff
@@ -378,5 +388,57 @@
       "text": "Finalizing",
       "value": "{'state': {'_eq': 'Finalizing'}}"
     }
+  ],
+  "tags": [
```

Comment:
Please create the translation keys.
https://github.com/gamedaoco/gamedao-haiku/blob/main/pkg/translations/src/app/en-US.json

Reply:
Done